packages/adapter-prisma/src/adapter-prisma.ts

```diff
@@ -1,26 +1,98 @@
 import pWaterfall from 'p-waterfall';
 import { defaultObj, mapKeys, identity, flatten } from '@keystone-next/utils-legacy';
 
+// Note: These type definitions are preliminary while we're working towards
+// a full TypeScript conversion.
+type Rel = {
+  left: Field<any>;
+  right: Field<any>;
+  cardinality: 'N:N' | 'N:1' | '1:N' | '1:1';
+  tableName: string;
+  columnName: string;
+};
+type Rels = Rel[];
+
+type ListAdapterConfig = { searchField?: string };
+
+type List = { adapter: PrismaListAdapter };
+
+type Field<P extends string> = {
+  isRelationship: boolean;
+  path: P;
+  listKey: string;
+  refListKey: string;
+  isPrimaryKey: boolean;
+  adapter: PrismaFieldAdapter<P>;
+};
+
+type ItemQueryArgs = Record<string, any>;
+
+type IdType = string | number;
+
+type FromType = { fromId?: IdType; fromField?: string; fromList?: List };
+
+type PrismaModel = {
+  count: (filter: Filter) => Promise<number>;
+  findMany: (filter: Filter) => Promise<any[]>;
+  delete: (arg: { where: { id: number } }) => Promise<void>;
+  findUnique: (args: {
+    where: { id: number };
+    include?: Record<string, any>;
+  }) => Promise<Record<string, any> | undefined>;
+  create: (args: { data: Record<string, any>; include?: Record<string, any> }) => Promise<any>;
+  update: (args: {
+    where: { id: number };
+    data: Record<string, any>;
+    include?: Record<string, any>;
+  }) => Promise<any>;
+};
+
+type Filter = {
+  where?: Record<string, any>;
+  take?: number;
+  skip?: number;
+  orderBy?: Record<string, any>;
+  include?: Record<string, boolean>;
+};
+
 class PrismaAdapter {
+  config: {
+    prismaClient?: any;
+    provider?: 'postgresql' | 'sqlite';
+    enableLogging?: boolean;
+    url?: string;
+  };
+  listAdapters: Record<string, PrismaListAdapter>;
+  listAdapterClass?: any;
+  _prismaClient?: any;
```

Comment:
Where is this used?

Reply:
Good catch, legacy from when branch was first made. Will remove 👍